gravitee/values.yaml

```diff
@@ -49,7 +53,7 @@ api:
   replicaCount: 1
   image:
     repository: graviteeio/management-api
-    tag: 1.15.0
+    tag: 1.19.3
```

Comment:
it should be of course the last version, still we should introduce (like in the case of official charts) a process to update the chart when a new release of gravitee is available.
this will allow to version the charts accordingly

Reply:
Do you think we can provide this chart to incubator repository for official charts ?
If you have any documentation (or better, if you want to take  care), please share.

Reply:
Ok get the pointer to doc: https://github.com/helm/charts/blob/master/CONTRIBUTING.md#how-to-contribute-a-chart

Reply:
sure it should be possible, i think the process requires to use a fork of the repo. we did that for another software, but i don't recall the whole process, i can say it takes time.

we can work on that the next time we update gravitee in production (it should be in november), so we test the chart properly.